xtask/src/ci/target_matrix.rs

```diff
@@ -158,6 +178,9 @@ impl TargetMatrixArgs {
             std::mem::take(matrix);
             return;
         }
+        if self.has_image {
```

Comment:
I don't believe this is used, ever, since it's never called from either `weekly.yml` or `ci.yml`.

Reply:
it is, via https://github.com/Emilgardis/cross/blob/0a4ff656c7881425aef62552d9e4ed13efd49d2e/xtask/src/ci/target_matrix.rs#L29